cognitedata/reveal
# style: Remove border from Reveal in example page

#### Type of change
![Style](https://img.shields.io/badge/Type-Style-blue) 

#### Jira ticket :blue_book:
No JIRA ticket

## Description :pencil:
This PR removes the glaring white border from Reveal on the example page. After this PR, the page looks like this:

![Skjermbilde 2022-12-14 kl  11 18 36](https://user-images.githubusercontent.com/7979907/207569922-3fdc221e-91d9-4ee7-bec1-4b8de285d9d4.png)

Since the axis view controller (not exactly sure what to call this) is on the bottom right _and_ the FPS now overlaps the Reveal container more, I decided to move the FPS meter to the bottom left instead. 

## How has this been tested? :mag:

By running the example page and opening `localhost:3000`. 


## Test instructions :information_source:

Run the example page locally and verify that the styling looks correct. More concretely I would look for:

- Does the Reveal container fill up the entire screen?
- Is some of the Reveal containers outside of the browser's viewport? (e.g. does scroll bars appear or similar)


## Checklist :ballot_box_with_check:
